### DISCUSSION TITLE
Workflow for debugging an alert

### DISCUSSION THREAD

**Discussion by emschwartz** [autometrics-dev/autometrics-shared#46] [Ideas]
When an alert for an SLO happens, you kind of want to go through the following steps. We should think about ways to make this exact flow easier:
1. Look at the error rate or latency graph for all the functions that comprise the SLO (you can do this in the SLO Grafana dashboard we provide)
2. If there's a specific function that appears to be causing the problem, you probably want to look at the functions that were called by that one to see if most of the higher-level issue is attributable to the inner function
3. From there, you probably want to look at the recent changes to that function, the source code, and the logs

We might be able to add some kind of links within the Grafana dashboard to enable you to click between the various dashboards easily 🤔

**Comment by IvanMerrill** [comment]
You could add a link within the IDE tool tip to show graphs for all functions called by this particular function, although that still involves switching between your IDE and the dashboards which isn't great either. It's certainly possible to add links to a dashboard (see [here](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/manage-dashboard-links/ )) but a dashboard doesn't lend itself to iterating through the functions without the user taking some action
